### PR TITLE
Changes to Invite Rewards banner

### LINKED ISSUE #405: UI changes to Send screen for inviting
### What would you like?
When sending, if the user is verified they should see a banner that tells them that they'll receive 1 cUSD if they invite someone.

### Why is this needed?
We're going to be sending people rewards for inviting others.

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Screen Shot 2021-05-25 at 23 17 02

See also https://github.com/celo-org/wallet/pull/411

Many small changes, all related to invite rewards:
- Changed the invites banner.
- Changed the logo icon for users in Valora to be all green.
- Removed `GetRewardPill`.

This banner is still hidden behind a feature flag.

### Other changes

N/A

### Tested

Unit tests and manually

### How others should test

Open the Send screen. If the user is verified they should see a banner that tells them about the invite rewards, otherwise a banner to confirm their account.

If the user presses the banner a WebView should be opened with the terms and conditions (page is still being built so it will 404 for now).

### Related issues

- Fixes #405

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/send/InviteRewardsBanner.tsx [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+  const rewardAmount = useSelector((state) => state.send.inviteRewardCusd)
+
+  const openInviteTerms = () => {
+    navigate(Screens.WebViewScreen, { uri: INVITE_REWARDS_TERMS_LINK })
+  }
 
   return (
-    <View style={styles.container} testID="InviteRewardsBanner">
-      <Invite />
+    <TouchableOpacity
```
Comment (jeanregisser) [inline packages/mobile/src/send/InviteRewardsBanner.tsx:23]: We should use `Touchable` instead so it uses the appropriate platform effect on iOS and Android,

**Review by jeanregisser — APPROVED** [review]
LGTM 👍

**Comment by Celoqa** [comment]
Hey @gnardini I have verified this issue using latest Android internal build and Test Flight 1.15.0 & observed following:
**Observations:**

- [ ] User is not able to see "Get $1 by referring a friend" banner on Send Contact screen even user has verified phone number & also have some balance in account
**FYI:** We have raise query ticket #448 for same
- [ ] User's are not able to see "V" symbol when user make transaction by searching phone number[Attachment](https://drive.google.com/file/d/1_s9pqz74RaGoxh7HyBhOqAAeEdK2cACs/view?usp=sharing)
- [x] User is able to see "V" symbol when user makes transaction by scanning QR code of same number (from above scenario) [Attachment](https://drive.google.com/file/d/1HpXQ8wy0q9ATo2G9lgmuzbTcHe95wPyo/view?usp=sharing) 
- [ ] Also, user is able to see "Earn" pills instead of "EarnReward" only for non-verified user (Not able to see for Non-verified user).  .We have raise query 447 for same 
- [x] Also it is observed that, search field filters the contacts correctly when we start t give inputs. And user redirect back to Send contact screen after tapping < on Send Amount page
**Verified On Devices:** Redmi Note 8 (9.0), iPhone SE (14.2)
Thanks..!!

**Comment by MuckT** [comment]
https://valora.testquality.com requirement [R23](https://valora.testquality.com/project/12128/testcase/286101/requirement/11356). Do not remove or edit.

**Pull request merged**